### ISSUE TITLE
Power Supply - Redundancy

### ISSUE THREAD

**Issue by lucanos** [jcl5m1/ventilator#15]
With regards to the desire to have this unit operate in the event of a mains power outage.

12v Car Battery
- Car Battery Trickle Charger
- Solar Charger

As I understand it, a car battery has a capacity of >45ah. In the event of a lockdown, sourcing a battery from a vehicle shouldn’t be too difficult.

Car battery chargers are a little more challenging to source than a USB power supply, but worst case scenario, the battery could be recharged from a vehicle (either just leaching power from the fitted battery, or connecting the CPAP battery back to the car and running it for a period of time, letting the alternator charge it back up).

Just an idea. Smarter people might refine it from here.

**Comment by jcl5m1** [comment]
Yes, I've tried to run my CPAP from a car battery before.  It works well.  The pump can draw around 5A.  Maybe 3A RMS, so it'll last around 10-15 hours.

**Comment by hobodrifterdavid** [comment]
To give an idea, a 7Ah 12V battery can be picked up for probably less than $10 in any village in the developping world. (Used for motorcycles).
